dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/json/DynamicSimulationParametersSerializer.java

```diff
@@ -32,6 +32,9 @@ public void serialize(DynamicSimulationParameters parameters, JsonGenerator json
         jsonGenerator.writeStringField("version", DynamicSimulationParameters.VERSION);
         jsonGenerator.writeNumberField("startTime", parameters.getStartTime());
         jsonGenerator.writeNumberField("stopTime", parameters.getStopTime());
+        if (parameters.getDebugDir() != null) {
+            jsonGenerator.writeStringField("debugDir", parameters.getDebugDir());
+        }
```

Comment:
You can use this, instead:
`JsonUtil.writeOptionalStringField(jsonGenerator, "debugDir", parameters.getDebugDir()); `

Reply:
DONE in https://github.com/powsybl/powsybl-core/pull/3455/commits/7c47658f6b28d0361b4d65f9616aa95e05ceb679